### PR TITLE
added ErrorPage (similar to under construction page and stories) for CRAIG issue-944

### PR DESCRIPTION
**Pull request by Ay1man2**

### CODE REVIEW

File: src/components/ErrorPage.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { WarningAlt } from "@carbon/icons-react";
+
+/**
+ * Under Construction Page
+ */
+const ErrorPage = () => {
```
Comment (evpirnia) [inline src/components/ErrorPage.js:7]: et's change this to PageNotFound

File: src/components/ErrorPage.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  return (
+    <div className="underConstruction flexDirectionColumn">
+      <WarningAlt size="128" />
+      <h4>Error 404: Page Not Found</h4>
```
Comment (evpirnia) [inline src/components/ErrorPage.js:11]: let's mimic IBM Cloud a bit here. 

change to "Unfortunately, we couldn’t find the page you were looking for, but here are some helpful places to start from:" then list links for Projects, About, and Tutorial. 

Screenshot 2023-10-19 at 3 52 08 PM

File: src/components/ErrorPage.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import { WarningAlt } from "@carbon/icons-react";
+
+/**
+ * Under Construction Page
+ */
+const ErrorPage = () => {
+  return (
+    <div className="underConstruction flexDirectionColumn">
+      <WarningAlt size="128" />
```
Comment (evpirnia) [inline src/components/ErrorPage.js:10]: can we try this with the carbon Help icon instead? `<Help />`

**Review by evpirnia — CHANGES_REQUESTED** [review]

**Comment by jvallexm** [comment]
closing, needs to be implemented in craig

**Pull request closed without merging**